bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
```

Comment:
@Makman2 Thanks for pointing that out :). Should I use the range function to get the range of the diff and then use the _get_changes function for parsing?